bucket/webcopy.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version": "1.8.0.652",
+    "description": "Copy websites locally for offline browsing",
+    "homepage": "https://www.cyotek.com/cyotek-webcopy",
+    "license": {
+        "identifier": "Unknown",
+        "url": "https://www.cyotek.com/cyotek-webcopy/license-agreement"
+    },
+    "url": "https://www.cyotek.com/downloads/get/setup-cyowcopy-1.8.0-build-652.exe",
+    "hash": "ab328d7823f66bd99038be854de76c2e3abac5684190852982d6aa4a3c46edaa",
+    "innosetup": true,
```

Comment:
Why do you ignore configuation in inno archive??

`userappdata\Cyotek\Shared` contains default useragents
`userappdata\Cyotek\common` contains useful mimetypes

![image](https://user-images.githubusercontent.com/13260377/86099047-3a699180-bab7-11ea-9441-6fed9c3bc26d.png)

Reply:
Well, before you told me, I didn't know anything about these file.

Reply:
I'll try to sovle this problem, thx.

Reply:
Have changed. Manually copy files in the `{userappdata}` to `$Env:APPDATA`. Since options are saving in the registry, no need to persist options.